package.json

```diff
@@ -10,8 +10,7 @@
     "prebuild": "npm run clean",
     "build": "gitbook build",
     "clean": "del _book",
-    "deploy": "npm run build && npm run deploy:rsync && git fetch",
```

Comment:
No idea, but before this PR only `npm run deploy:rsync` was executed by CIRCLE, so `npm run build` was not called either 🤔